OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -72,7 +72,10 @@ public void InvokeCommand(string name, string arg)
 				case "givecash":
 					var givecashorder = new Order("DevGiveCash", world.LocalPlayer.PlayerActor, false);
 					int cash;
-					int.TryParse(arg, out cash);
+					float fcash;
+
+					float.TryParse(arg, out fcash);
+					cash = (int) fcash;
```

Comment:
This can be simplified one step further by redefining cash above to a float, and then casting directly from the float to uint when assigning to the ExtraData.

Reply:
Simplified the float cast as requested: 8fa9fade113371d82141bf1dec3879fe1a23ed5d